cfy_manager/components/cli/cli.py

```diff
@@ -52,34 +52,25 @@ def _set_colors(is_root):
 def _configure():
     username = config[MANAGER][SECURITY]['admin_username']
     password = config[MANAGER][SECURITY]['admin_password']
+    ssl_enabled = config[MANAGER][SECURITY]['ssl_enabled']
 
     use_cmd = ['cfy', 'profiles', 'use',
```

Comment:
I don't know why originally there were two commands running  (`use` and then `set`). I can't think of a good reason to do so. I combined them into one command.

Reply:
because of #26